### PR TITLE
fix(styles): message popover adjustments

### PR DESCRIPTION
**Pull request by N1XUS**
relates #4912

- Adjusted message popover parameters for arrow icon font size;
- List item paddings;
- List item title font metrics

### CODE REVIEW

File: packages/styles/src/mixins/list/_list-message-view.scss [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 .#{$block} {
   &--message-view {
+    --faListNavigationIndicatorFontSize: 0.75rem;
```
Comment (mikerodonnell89) [inline packages/styles/src/mixins/list/_list-message-view.scss:5]: `--fa`? Also there's a variable, not sure how important it is to use but `--sapFontSmallSize`

**Review by mikerodonnell89 — CHANGES_REQUESTED** [review]
see my comment above

File: packages/styles/src/mixins/list/_list-message-view.scss [hunk 2]
```diff
@@ -40,7 +41,14 @@
       @include fd-ellipsis();
     }
 
+    .fd-list__item {
+      height: auto;
+    }
```
Comment (InnaAtanasova) [inline packages/styles/src/mixins/list/_list-message-view.scss:46]: `.fd-list` should be `.#{$block}`

File: packages/styles/src/mixins/list/_list-message-view.scss (hunk 1 above)
Comment (InnaAtanasova) [inline packages/styles/src/mixins/list/_list-message-view.scss:5]: typo, should be `--fd` not `--fa`

File: packages/styles/src/mixins/list/_list-message-view.scss [hunk 3]
```diff
@@ -40,7 +41,14 @@
...
+
     .#{$block}__link {
+      padding-top: 0.5rem;
+      padding-bottom: 0.5rem;
```
Comment (InnaAtanasova) [inline packages/styles/src/mixins/list/_list-message-view.scss:50]: you can use the `fd-set-paddings-y-equal` mixin

**Review by InnaAtanasova — CHANGES_REQUESTED** [review]

**Pull request merged**